### DISCUSSION TITLE
Uploading to storage buckets only gets the error message 'Internal server error'

### DISCUSSION THREAD

**Discussion by e11sa** [supabase/supabase#21318] [Questions]
Hello, community! Have someone faced the similiar issue? Recently our storage uploading stopped working. 
Uploading to storage buckets only gets the error message: 'Internal server error' and cannot upload anything now. We have removed all policies but it gives the same error. Also no errors are visible in logs.

**Comment by GaryAustin1** [comment]
Does it work from the dashboard UI?  Can you read files?
Restart your instance.  Reset your browser local storage and flush cache if on just one browser.

Removing policies would block access but you should get RLS violation error.

**Comment by Sir-Baconn** [comment]
I have the same issue, dashboard UI does not load buckets with console filled with 500s 
![image](https://github.com/supabase/supabase/assets/11286486/114ba29a-fa65-45d1-8e7f-fbd2b4f5756a)

If I try to create a new bucket I also receive a 500
![image](https://github.com/supabase/supabase/assets/11286486/385549a9-ce9e-47eb-a092-5afe902cbe74)

Project restart did not work.

Were you able to fix this?

**Comment by BruceSuperProgramer** [comment]
I was working through the Supabase AI tutorial project( [ChatGaptYourFiles](https://github.com/supabase-community/chatgpt-your-files)) and initially used the main branch. Before pushing the migration to the remote repository, I created a bucket for testing purposes, and the file upload process worked smoothly. However, after deleting the bucket and successfully pushing the migration from the codebase to the remote, I encountered an internal error message when attempting to upload files to the bucket, both from the dashboard and the Next.js app.

**Reply by BruceSuperProgramer** [reply]
Essentially, the issues arose immediately after the application of one of the migration files.

**Reply by BruceSuperProgramer** [reply]
Subsequently, even when I created a new bucket and attempted to upload a file, I encountered the same error.
